hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -1704,7 +1704,7 @@ case class TableParallelize(rowsAndGlobal: IR, nPartitions: Option[Int] = None)
 
     val rowsT = ptype.types(0).asInstanceOf[PArray]
     val rowT = rowsT.elementType.asInstanceOf[PStruct].setRequired(true)
-    val spec = TypedCodecSpec(rowT, BufferSpec.wireSpec)
+    val spec = TypedCodecSpec(rowT, BufferSpec.blockedUncompressed)
```

Comment:
Was this the key change that restored parity?

Reply:
No, these ones were fine.

Reply:
Will revert this